docs/server-setup.md

```
@@ -0,0 +1,51 @@
+
+`ServerSetupPlugin` bootstraps a server's structure from a preset template: categories, text and voice channels, roles, role-level permissions, and per-channel permission overwrites (like staff-only areas).
+
+```python
+from easycord.plugins import ServerSetupPlugin
+
+bot.add_plugin(ServerSetupPlugin())
+```
+
+## The `/setup-server` command
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)